packages/conntrack-tools/build

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
`conntrack-tools` contains `conntrack` command required by felix component to remove conntrack entries after workloads are deleted.